go/chat/supersedes.go

```diff
@@ -179,7 +180,16 @@ func (t *basicSupersedesTransform) Run(ctx context.Context,
 				// superseded by anything.  Could have been deleted by a
 				// delete-history, retention expunge, or was an exploding
 				// message.
-				continue
+				if newMsg.IsValid() {
+					// If we want to show the GUI that the message is exploded,
+					// don't hide these yet
+					mvalid := newMsg.Valid()
+					if !mvalid.IsExploding() || mvalid.HideExplosion(time.Now()) {
+						continue
+					}
+				} else {
+					continue
+				}
```

Comment:
I'll ask you about this part in person to make sure I understand :)

Reply:
resolved offline -- we don't want to filter out exploded messages until we should be 'hiding' the explosion